src/Service/NaiveHtmlCrawlerManager.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the "StenopePHP/Stenope" bundle.
+ *
+ * @author Thomas Jarrand <thomas.jarrand@gmail.com>
+ */
+
+namespace Stenope\Bundle\Service;
+
+use Stenope\Bundle\Behaviour\HtmlCrawlerManagerInterface;
+use Stenope\Bundle\Content;
+use Symfony\Component\DomCrawler\Crawler;
+
+class NaiveHtmlCrawlerManager implements HtmlCrawlerManagerInterface
+{
+    private array $crawlers = [];
+
+    public function get(Content $content, array &$data, string $property): ?Crawler
+    {
+        $key = "{$content->getType()}:{$content->getSlug()}";
+        $crawler = $this->createCrawler($data[$property]);
+
+        if ($crawler) {
+            if (!isset($this->crawlers[$key])) {
+                $this->crawlers[$key] = [];
+            }
+
+            $this->crawlers[$key][$property] = $crawler;
+        }
+
+        return $crawler;
+    }
+
+    public function save(Content $content, array &$data, string $property, bool $force = false): void
```

Comment:
`$force` is not used for this implementation. Do we even need `$force` in the interface actually? What is the use-case ? I think this is an implementation detail of the shared crawler.

Reply:
Yes, I felt like I needed to add a safety net for edge-case where you needed to force the persistance of current changes before the loop is over. But maybe it's useless.
And yes, it only makes sens for the SharedHTMLCrawler.